lib/ws_servers/algos/index.js

```diff
@@ -39,9 +43,11 @@ module.exports = class AlgoServer extends WSServer {
     this.wsURL = wsURL
     this.restURL = restURL
     this.apiDB = apiDB
+    this.algoDB = algoDB
     this.dbPath = hfLowDBPath
     this.clients = {}
     this.hosts = {}
+    this.aoc = new AlgoServerClient({ url: `http://localhost:${port}` })
```

Comment:
wait, is this required? the algo orders server sets up a client? why a client-server model then?